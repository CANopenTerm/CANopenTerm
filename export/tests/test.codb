1000::Device Type:mandatory::VAR:m:UNSIGNED32:m:ro:d[const,ro]::n::n:::::::n:m

$1001=1000

1003::Predefined Error Field:optional::ARRAY:m::m::m:2:m:255:m::m::m::m::m
1003:00:Number of Errors:optional::VAR:m:UNSIGNED8:m:rw:d[rw,ro]::n::n:::::0:m:n:m
1003:01:Standard Error Field:optional::VAR:m:UNSIGNED32:m:ro:m::n::n::::d:0:m:n:m

$1004=1003
