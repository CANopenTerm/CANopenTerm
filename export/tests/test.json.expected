[{"id":"1000","index":4096,"sub_indices":[{"index":0,"desc":"Device Type","kind":1,"data_type":{"type":9,"attr":0,"lower":0,"upper":0},"access_type":{"type":2,"attr":0,"lower":0,"upper":0},"low_limit":{"value":0,"attr":0,"lower":0,"upper":0},"high_limit":{"value":0,"attr":0,"lower":0,"upper":0},"default_value":{"value":0,"attr":0,"lower":0,"upper":0},"mappable":{"value":false,"attr":0,"lower":0,"upper":0}}],"code":{"type":3,"attr":1,"lower":0,"upper":0},"min_elements":{"value":0,"attr":0,"lower":0,"upper":0},"max_elements":{"value":0,"attr":0,"lower":0,"upper":0},"desc":"Device Type","kind":1},{"id":"1001","index":4097,"sub_indices":[{"index":0,"desc":"Device Type","kind":1,"data_type":{"type":9,"attr":0,"lower":0,"upper":0},"access_type":{"type":2,"attr":0,"lower":0,"upper":0},"low_limit":{"value":0,"attr":0,"lower":0,"upper":0},"high_limit":{"value":0,"attr":0,"lower":0,"upper":0},"default_value":{"value":0,"attr":0,"lower":0,"upper":0},"mappable":{"value":false,"attr":0,"lower":0,"upper":0}}],"code":{"type":3,"attr":1,"lower":0,"upper":0},"min_elements":{"value":0,"attr":0,"lower":0,"upper":0},"max_elements":{"value":0,"attr":0,"lower":0,"upper":0},"desc":"Device Type","kind":1},{"id":"1003","index":4099,"sub_indices":[{"index":0,"desc":"Number of Errors","kind":0,"data_type":{"type":3,"attr":0,"lower":0,"upper":0},"access_type":{"type":4,"attr":0,"lower":0,"upper":0},"low_limit":{"value":0,"attr":0,"lower":0,"upper":0},"high_limit":{"value":0,"attr":0,"lower":0,"upper":0},"default_value":{"value":0,"attr":0,"lower":0,"upper":0},"mappable":{"value":false,"attr":0,"lower":0,"upper":0}},{"index":1,"desc":"Standard Error Field","kind":0,"data_type":{"type":9,"attr":0,"lower":0,"upper":0},"access_type":{"type":2,"attr":0,"lower":0,"upper":0},"low_limit":{"value":0,"attr":0,"lower":0,"upper":0},"high_limit":{"value":0,"attr":0,"lower":0,"upper":0},"default_value":{"value":0,"attr":0,"lower":0,"upper":0},"mappable":{"value":false,"attr":0,"lower":0,"upper":0}}],"code":{"type":4,"attr":1,"lower":0,"upper":0},"min_elements":{"value":2,"attr":0,"lower":0,"upper":0},"max_elements":{"value":255,"attr":0,"lower":0,"upper":0},"desc":"Predefined Error Field","kind":0},{"id":"1004","index":4100,"sub_indices":[{"index":0,"desc":"Number of Errors","kind":0,"data_type":{"type":3,"attr":0,"lower":0,"upper":0},"access_type":{"type":4,"attr":0,"lower":0,"upper":0},"low_limit":{"value":0,"attr":0,"lower":0,"upper":0},"high_limit":{"value":0,"attr":0,"lower":0,"upper":0},"default_value":{"value":0,"attr":0,"lower":0,"upper":0},"mappable":{"value":false,"attr":0,"lower":0,"upper":0}},{"index":1,"desc":"Standard Error Field","kind":0,"data_type":{"type":9,"attr":0,"lower":0,"upper":0},"access_type":{"type":2,"attr":0,"lower":0,"upper":0},"low_limit":{"value":0,"attr":0,"lower":0,"upper":0},"high_limit":{"value":0,"attr":0,"lower":0,"upper":0},"default_value":{"value":0,"attr":0,"lower":0,"upper":0},"mappable":{"value":false,"attr":0,"lower":0,"upper":0}}],"code":{"type":4,"attr":1,"lower":0,"upper":0},"min_elements":{"value":2,"attr":0,"lower":0,"upper":0},"max_elements":{"value":255,"attr":0,"lower":0,"upper":0},"desc":"Predefined Error Field","kind":0}]
